src/vsr/clock.zig

```diff
@@ -769,7 +786,10 @@ const ClockSimulator = struct {
 };
 
 test "clock: fuzz test" {
-    std.testing.log_level = .err; // silence all clock logs
+    // Silence all clock logs.
+    var level = std.log.Level.err;
```

Comment:
Ah, this was a mistake in the code snippet I sent! It should be 
```zig
var level = std.testing.log_level;
```

Reply:
Also can be const. 

I wonder if we should add somthing akin to Rust's `std::mem::replace`?

```zig
    const level = stdx.swap_with(&std.testing.log_level, std.log.Level.err);
    defer std.testing.log_level = level;
```

We have 

```zig
const foo = bar;
bar = baz
```

logic fairly often, and my eyes skip over it automatically, so abstracting it away _might_ reduce trivial bugs. 

OTOH, I feel like the zig's way would be not to wrap such trivial constructs...